src/allocator.h

```diff
@@ -181,6 +184,12 @@ allocator_get_capacity(allocator *al)
    return al->ops->get_capacity(al);
 }
 
+static inline uint64
+allocator_get_extent_num(allocator *al, uint64 page_addr)
+{
+   return al->ops->get_extent_num(al, page_addr);
```

Comment:
Externalizing this as virtual fn so we can print this value in `clockcache.c` when the assertion trips.